project/src/components/map/map.tsx

```diff
@@ -2,18 +2,18 @@ import {useRef, useEffect} from 'react';
 import leaflet from 'leaflet';
 import 'leaflet/dist/leaflet.css';
 import useMap from '../../hooks/use-map/use-map';
-import {City, Location} from '../../types/offers';
+import {Offers} from '../../types/offers';
 import '../../const';
 import { URL_MARKER_DEFAULT } from '../../const';
 
 type MapProps = {
-  city: City;
-  points: Location[];
+  offers: Offers;
 };
 
-function Map({city, points}: MapProps): JSX.Element {
+function Map({offers}: MapProps): JSX.Element {
+  const activeCity = (offers.length) ? offers[0].city : null;
   const mapRef = useRef(null);
-  const map = useMap(mapRef, city);
+  const mapObject = useMap(mapRef, activeCity);
```

Comment:
название переменной не должно включать ее тип. корректней было все же прежнее название. или как-то наподобие `mapItem`
```suggestion
  const mapItem = useMap(mapRef, activeCity);
```